examples/ibm-configuration-aggregator/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    ibm = {
+      source  = "terraform.local/ibm-cloud/ibm"
+      version = "0.0.1"
```

Comment:
Version is wrong. Use the version in which this feature will be available . Or remove this field completely.